__tests__/core/seekExportDefaultLine.js

```diff
@@ -2,37 +2,43 @@ const path = require('path');
 const fs = require('fs');
 const seekExportDefaultLine = require('../../lib/core/seekExportDefaultLine');
 
-const readComponent = (component, cb) => {
-  const filename = path.join(__dirname, `__fixtures__/methods/${component}.vue`);
+const readComponent = (componentPath, cb) => {
+  const filename = path.join(__dirname, `__fixtures__/methods/${componentPath}`);
 
   fs.readFile(filename, 'utf8', (err, source) => cb(source));
 };
 
 describe('core.seekExportDefaultLine', () => {
   test('A normal component', (done) => {
-    readComponent('Component', (source) => {
+    readComponent('Component.vue', (source) => {
       expect(seekExportDefaultLine(source)).toBe(9);
       done();
     });
   });
 
   test('A normal component with a lot of spaces', (done) => {
-    readComponent('ComponentWithSpaces', (source) => {
+    readComponent('ComponentWithSpaces.vue', (source) => {
       expect(seekExportDefaultLine(source)).toBe(14);
       done();
     });
   });
 
   test('When <script> is before <template>', (done) => {
-    readComponent('ScriptBeforeTemplate', (source) => {
+    readComponent('ScriptBeforeTemplate.vue', (source) => {
       expect(seekExportDefaultLine(source)).toBe(5);
       done();
     });
   });
   test('When `export default` is inside <template>', (done) => {
-    readComponent('ComponentWithExportDefaultInTemplate', (source) => {
+    readComponent('ComponentWithExportDefaultInTemplate.vue', (source) => {
       expect(seekExportDefaultLine(source)).toBe(6);
       done();
     });
   });
+  test('When component is js file', (done) => {
+    readComponent('Component/Component.js', (source) => {
+      expect(seekExportDefaultLine(source)).toBe(0);
```

Comment:
This is wrong, the line number should be 6 here.
You should update your component by removing `(` from `export default ({`, to be honest I've never seen this notation ^^

Reply:
true ! doing that.